engines/bladerunner/detection_tables.h

```diff
@@ -118,7 +118,7 @@ static const ADGameDescription gameDescriptions[] = {
 		AD_ENTRY1s("STARTUP.MIX", "5643b53306ca7764cf1ec7b79c9630a3", 2312374),
 		Common::EN_ANY,
 		Common::kPlatformWindows,
-		ADGF_NO_FLAGS,
+		ADGF_TESTING,
```

Comment:
actually this final/enhanced version is not yet ready for testing

Reply:
You might instead want to add the missing ADGF_UNSTABLE flag for this one and the other entries not yet ready for testing.